### PR TITLE
Updated readme to include fedora commands

### PR DESCRIPTION
**Pull request by liamwarfield**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Setup the application:
 
     $ cp development.ini.sample development.ini
     $ gearbox setup-app
+`If gearbox setup-app fails try specifying pip3 in the last two steps`
```
Comment (sumnerevans) [inline README.md]: Please only use "`"s to denote code blocks:

    If `gearbox setup-app` fails, try specifying `pip3` in the last two steps.

Also, note the comma after "fails".

File: README.md [hunk 2]
```diff
@@ -48,6 +49,9 @@ Make sure you have `virtualenvwrapper` installed
 On Arch Linux:
 
     $ sudo pacman -S python-virtualenvwrapper
+On Fedora
```
Comment (sumnerevans) [inline README.md]: Add a newline above this.

File: README.md [hunk 3]
```diff
@@ -70,6 +74,7 @@ Setup the application:
     $ cp development.ini.sample development.ini
     $ gearbox setup-app
 
+
```
Comment (sumnerevans) [inline README.md]: Remove this newline.

**Comment by jackrosenthal** [comment]
README needs converted to reStructuredText as well... does someone want to do this while making these changes?

**Comment by sumnerevans** [comment]
Why rst over md? (I don't care, just wondering what advantages it provides.)

**Comment by jackrosenthal** [comment]
See ``CONTRIBUITING.rst`` for why it's **required** for this project.

As for why reStructuredText is more capable than Markdown out of the box, consider the extra features it supports without needing to "flavor" it:

* Tables
* Footnotes
* Roles
* Directives
* Sidebars
* Topics
* File inclusion
* Rubrics
* Mathematics
* Admonitions
* Line blocks
* Parsed literal blocks
* Substitutions
* ...

See a short rant here (http://inside.mines.edu/~jrosenth/hacking-docutils.html) for why I consider reStructuredText better than Markdown for extensibility reasons.

**Comment by liamwarfield** [comment]
Maybe we could just specify pip3 in the original instructions to remove the need for the foot note.

**Comment by sumnerevans** [comment]
I want to give my philosophy on what a README should contain. The goal of a README should be to describe any information that is *directly important to the project* so that a person can quickly get all the information they need to get the project running or deployed. Basically, assume competence in the technologies used in the project, but do not assume knowledge *of the project itself*. (A section with links to external resources that can help the person become competent in the technologies is also useful, but should not detriment the usability of the README by people who already understand them.)

For example, we should assume competence in Python, but we should explain that this project uses Python 3 instead of Python 2 and that it is built on the TurboGears framework. We should assume competence in using a CLI and git, but we should not assume competence in using the `gearbox` utility.

This is what I expect a README to contain:

1. **An explanation of what the project is.** In our case, the explanation of what the project is should be a description of the project and a link to the ACM website. This should be no more than a couple of lines. (Again, this should not get in the way of someone just trying to figure out how to get the thing running.)

2. **A brief overview of how to deploy the project.** Eventually, we should describe how to deploy this to a server so that other clubs which do not have Jack as the President know how to deploy. Again, this should be a high-level overview of the requirements and procedure, it should not, for example, describe how to SSH to a server. (@jackrosenthal should probably be the one to write this section of the README.)

3. *A description of how to begin working on the project.** In our case, this would include how to install the necessary dependencies using `pip`, and how to ues `gearbox` to run the application locally. I actually do not think that it is useful to include how to use `virtualenv` is useful here. We can reference an article with information on how to use it, but since using a `virtualenv` is not *directly important to the project*, it should not be included.

If any of these sections becomes too long, we should reconsider that section's content and remove any unnecessary cruft. If all else fails, we can evaluate whether or not to split it out to another file.

In conclusion, after reading the README, the user should understand what the project does, be able to run the application locally, and deploy the project to production. Any further documentation (for example a list of all of the config options) should be in seprate files, linked from this document.

If you have differing opinions, please let me know. (Sorry for the essay.)

CC: @jackrosenthal , @samsartor
